tests/devices/qubit_mixed/test_qubit_mixed_measure.py

```diff
@@ -488,6 +439,7 @@ def test_autograd_backprop(self, coeffs):
         expected_gradient = qml.grad(self.expected)(x, coeffs)
         assert qml.math.allclose(expected_gradient, gradient)
 
+    @pytest.mark.skip("Implementation of csr not differentiable for autograd here")
```

Comment:
Are we skipping all the other tests in this file marked with `@pytest.mark.skip` for the same reason?

I guess this didn't happen with the legacy device implementation because it didn't use the `csr_matrix`. Is that so?

Reply:
you are right. Legacy mixed converts everything back to full and uses it's only methods there

Reply:
Thanks. Can we please write the same message in the other tests as a future reference?

Reply:
Not really. These tests are new from previous legacy ones. I'm not sure if the same things would even pass for legacy ones. I can look deeper into them later to summarize what's the real reason

Reply:
It's probably worth checking right now whether or not these tests pass with the old `default.mixed` or not (you can do that on `master`. If not, then we _definitely_ have a feature regression that needs to be fixed.

Reply:
You are right this is a bug. I don't even need to use old default.mixed because this same test is also in master and it works very well. I'll fix this tongiht

Reply:
Debugged at https://github.com/PennyLaneAI/pennylane/pull/6684/commits/a0a1fc8064ba821ff4f6d8c31c601fbf232028ad